frontend/src/components/RegisterEmployerLast/index.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import SRegisterEmployer from "./style";
+
+export default function RegisterEmployer() {
+  const [formData, setFormData] = useState({
+    nin: "",
+    categoryJur: "",
+    siret: "",
+  });
+
+  const hChangeFormData = (evt) => {
+    const newData = { ...formData };
+    newData[evt.target.name] = evt.target.value;
+    setFormData(newData);
+  };
+
+  const hSubmit = (evt) => {
+    evt.preventDefault();
```

Comment:
Don't you think it's missing something here ? 
I should refuse because you'll introducing useless UI elements on the website -_-